### PR TITLE
W-13658383-Add-limitation-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**

### CODE REVIEW

File: modules/ROOT/pages/test-recorder.adoc [hunk 1]
```diff
@@ -30,15 +30,15 @@ The following table describes the product versions required to use the test reco
...
+* Your validations fail every time your test runs if you configure `spy` or `assert` processors to assert values for random data, time-dependent information (such as timestamps), or values resulting from parallel processes, because those values change in every execution.
 * Mocking values resulting from parallel processes causes a mixture of real and mocked data that compromises the execution of the processors that follow in your test.
 * Although the recorder supports data iteration in the flow, such as recursions or loops, it does not support cases in which the structure of the data being tested changes inside the iteration.
-* The recorder does not support mocking a message before or inside a Foreach processor.
-
+* The recorder does not support mocking a message before or inside a `foreach` processor.
+* If you activate a test over a flow, you can mock parameters of the main flow and flows related to your main flow. MUnit Test Recorder ignores third-party flows and flows that are not related to your main flow.
```
Comment (aguozden) [inline modules/ROOT/pages/test-recorder.adoc]: Hi, I would describe the Test recorder limitation like this. Please feel free to correct the sentence if needed: "When using the Test Recorder to generate a test over a flow, you can only mock parameters included in that flow. The Test Recorder UI does not allow the user to mock parameters in sub-flows, not even referenced flows. Should you need to mock parameters in sub-flows of any other process in the project, you can modify the mocking component once the test has been created."

**Review by aguozden — CHANGES_REQUESTED** [review]
Hi Cristian, I have suggested a different description for the Test Recorder limitation

File: modules/ROOT/pages/test-recorder.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/test-recorder.adoc]: You should think about becoming a technical writer! hahaha Thanks!

**Review by mpereyra-salesforce — DISMISSED** [review]
LGTM!!!

**Review by aguozden — DISMISSED** [review]
Great!

File: modules/ROOT/pages/test-recorder.adoc [hunk 2]
```diff
@@ -30,15 +30,15 @@ The following table describes the product versions required to use the test reco
...
+* Your validations fail every time your test runs if you configure `spy` or `assert` processors to assert values for random data, time-dependent information (such as timestamps), or values resulting from parallel processes, because those values change in every execution.
 * Mocking values resulting from parallel processes causes a mixture of real and mocked data that compromises the execution of the processors that follow in your test.
 * Although the recorder supports data iteration in the flow, such as recursions or loops, it does not support cases in which the structure of the data being tested changes inside the iteration.
-* The recorder does not support mocking a message before or inside a Foreach processor.
-
+* The recorder does not support mocking a message before or inside a `foreach` processor.
+* When using the Test Recorder to generate a test over your flow, you can only mock parameters included in your flow. You cannot mock parameters in sub-flows or in referenced flows through the Test Recorder UI. If you need to mock parameters in sub-flows of any other process in the project, you can modify the mocking component after creating the test.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/test-recorder.adoc:41]: You use "test recorder" above, but "Test Recorder" here, is the capitalization intentional?

**Review by glenn-rodgers-sf — APPROVED** [review]

File: modules/ROOT/pages/test-recorder.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/test-recorder.adoc:41]: Great catch! Thanks!

**Pull request merged**